src/main/java/by/yahor_kulesh/repositories/TicketRepository.java

```diff
@@ -0,0 +1,22 @@
+package by.yahor_kulesh.repositories;
+
+import by.yahor_kulesh.entity.TicketEntity;
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+import java.util.UUID;
+
+@Repository
+public interface TicketRepository extends CrudRepository<TicketEntity, UUID> {
+
+    default void insertOrUpdateTicket(TicketEntity ticket){
+        this.save(ticket);
+    }
```

Comment:
you do not need this method. you use just `save` method:
1. If the entity has an ID that already exists in the database, the save method will update the existing record with any new field values.
2. If the entity does not have an existing ID (or has a new one), save will treat it as a new entity and insert it as a new record in the database.

Reply:
I thought, that i cannot use already declared methods, only wrapped ones